spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -165,6 +177,41 @@ public void setInitialSegmentCount(int initialSegmentCount) {
             this.initialSegmentCount = initialSegmentCount;
         }
 
+        /**
+         * See {@link TrackingEventProcessorConfiguration#andTokenClaimInterval(long, java.util.concurrent.TimeUnit)}
```

Comment:
I do not think this javadoc is sufficient to explain what you're returning, thus please be more specific here.

The same holds for the `@return` line.
Yes you're returning the `tokenClaimInterval`, but the sentence should be descriptive instead, something like:
`@return a {@code long} specifying the token claim interval used by a {@link TrackingEventProcessor}`

Reply:
The javadoc comments hold for all the added functionality in this class by the way.

Reply:
I do agree. Done.

Reply:
I do agree. Done.